components/dashboard/components/SectionHeader.jsx

```diff
@@ -9,7 +9,6 @@ const Title = styled.div`
 font-size: ${(props) => {
         return props.mainHeader ? '40px' : '32px';
     }};
-font-weight: bold;
```

Comment:
rebase artifact?

Reply:
This might be for the the new design for the dashboard.